recipes/spleeter/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "spleeter" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: fea6b1748e285cdef564d439f14500ed4a6c5ce8721f440e1dd7ae656749aaae
+  - url: https://pypi.io/packages/source/n/norbert/norbert-0.2.1.tar.gz
+    sha256: bd4cbc2527f0550b81bf4265c1a64b352cab7f71e4e3c823d30b71a7368de74e
+  - url: https://pypi.io/packages/source/m/musdb/musdb-0.3.1.tar.gz
+    sha256: 27aac71095f919c7e0cfb526af1e6f745275645e959f759c42711354ea0fcbf7
+  - url: https://pypi.io/packages/source/m/museval/museval-0.3.0.tar.gz
+    sha256: 2e1334dd3367dea562c1902515f6d55e87dc3072cfc411c616ac6f0a48d31d5f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ffmpeg
+    - requests
+    - setuptools
+    - importlib_resources  # [py<37]
+    - tensorflow ==1.14.0
+    - pandas ==0.25.1
+
+test:
+  imports:
+    - spleeter
+    - spleeter.commands
+    - spleeter.model
+    - spleeter.utils
+    - spleeter.separator
+
+about:
+  home: https://github.com/deezer/spleeter
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'The Deezer source separation library with pretrained models based on tensorflow.'
+  doc_url: https://github.com/deezer/spleeter/wiki
+  dev_url: https://github.com/deezer/spleeter
+
+extra:
+  recipe-maintainers:
+    - deezer
```

Comment:
Deezer is a GitHub organisation so I don't think it can be a maintainer. Have you asked anyone from it? It might be best to add a few people directly instead.

Reply:
Oh, didn't know that. Yes, @romi1502 and I are employees from Deezer company and are releasing our open source library that we would like to provide Conda package for. Since we are belonging to the company and this package would be an official company release with repository in the Deezer organisation I though it should be added as maintainer. We remove it then